docs/kb/api-version-4/export.md

```diff
@@ -0,0 +1,352 @@
+
+OR
+
+* “Admin > Exports > View”
+
+### Arguments
+
+* **created_after**: Selects visitor activities that were created after the specified time. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+* **created_before**: (Optional) Selects visitor activities that were created before the specified time. This value must be after the value in `created_after`. If this argument is not specified, then no upper boundary is used in the query, and all data after the `created_after` is returned. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+
+__NOTE:__ The range between created_after and created_before cannot exceed 1 year. When created_before is not specified, the current date is used to gauge the interval.
+
+# Using the Export API
+
+---
+
+# Create
+
+```
+/api/export/version/4/do/create
+```
+
+Used to create an export of an object and procedure.
+
+## POST
+
+### Request Body
+
+Content-Type must be `application/json`.
+
+Input Representation
+
+* **object**: Specifies which object to export. Currently only "visitorActivity" is supported.
+* **procedure**: The procedure to execute. A procedure is a query and execution plan used to retrieve data. Each object has a different set of procedures. See the [Procedures](#procedures) section for available procedures.
+    * **name**: The name of the procedure.
+    * **arguments**: Arguments used to manipulate the behavior of the procedure. These arguments are specific to the procedure. See the documentation for the procedure to see which arguments apply.
+
+```json
+{
+	"object": string,
+	"procedure": {
+		"name": string,
+		"arguments": {
+			"argument name": argument value,
+			// additional arguments...
+		}
+	}
+}
+```
+
+### Success
+
+Status Code: 200
+
+Output Representation
+
+* **id**: The ID of the export. ID is used to check the status of the export.
+* **state**: The state of the export. Displays "Waiting" when the export has been queued for processing, "Processing" when the server is working on the export and "Complete" when the export has completed. See [Export State](#export-state) enum.
+* **isExpired**: Indicates that the export has expired. When an export has expired, this will return `true` and no data associated to the export can be downloaded.
+* **resultsRef**: (Optional) This property appears when an export is complete, and it contains a list of URLS to CSV files available for download. If there is no data associated with the export, this property is absent. If there is only a single CSV file available for download, this property is a string containing the URL to download the file. If there are multiple CSV files, resultRef is an array of URLs.
```

Comment:
There's some inconsistency regarding `resultRefs`. In the example responses, it's almost always spelled `resultsRef`, and here, it needs to be pluralized.